src/inmanta/server/server.py

```diff
@@ -80,18 +81,15 @@ def check_storage(self) -> dict[str, str]:
         Check if the server storage is configured and ready to use.
         """
 
-        def _ensure_directory_exist(directory: str, *subdirs: str) -> str:
-            directory = os.path.join(directory, *subdirs)
-            if not os.path.exists(directory):
-                os.mkdir(directory)
-            return directory
-
         state_dir = config.state_dir.get()
         server_state_dir = os.path.join(state_dir, "server")
-        dir_map = {"server": _ensure_directory_exist(state_dir, "server")}
-        dir_map["environments"] = _ensure_directory_exist(server_state_dir, "environments")
-        dir_map["agents"] = _ensure_directory_exist(server_state_dir, "agents")
-        dir_map["logs"] = _ensure_directory_exist(config.log_dir.get())
+        dir_map = {
+            "server": ensure_directory_exist(state_dir, "server"),
+            "environments": ensure_directory_exist(server_state_dir, "environments"),
+            "agents": ensure_directory_exist(server_state_dir, "agents"),
```

Comment:
Will this folder still exist after we switch to the Scheduler? I don't see it in your new layout (although it might be for compatibility reasons)

Reply:
No indeed it should be removed at some point but we have to keep it until the old agent is stripped out.